src/components/AsideHeader/__stories__/AsideHeaderShowcase.tsx

```diff
@@ -35,7 +35,20 @@ export function AsideHeaderShowcase() {
                     href: '#',
                     onClick: () => alert('click on logo'),
                 }}
-                menuItems={menuItemsShowcase}
+                menuItems={[
+                    ...menuItemsShowcase,
+                    {
+                        id: 'search',
```

Comment:
The same id has already been used in `subheaderItems`)
Let's rename this item id and add additional panel to menuItems